src/data/content/risorse-per-progettare/realizzare/sviluppo-interfaccia.yaml

```diff
@@ -31,7 +31,7 @@ metadata:
 # ————————————————————————————————————————————
 seo:
   name: Sviluppo interfaccia - Designers Italia #C REMIND "- Designers Italia" #AUTO
-  description: "Realizza il front-end di un servizio digitale con codice e componenti open source di sviluppo" #C
+  description: "Realizza la parte front-end di un servizio digitale con codice e componenti di sviluppo open source." #C
```

Comment:
```suggestion
  description: "Realizza l'interfaccia di un servizio digitale con codice e componenti di sviluppo open source." #C
```

Reply:
Su questo non ho un'opinione precisa; se non erro, prima era già scritto "interfaccia" ed è stato cambiato in "il front-end".